### PR TITLE
Fixed error (500) response on not allowed filter

### PR DESCRIPTION
**Pull request by Josca**

### CODE REVIEW

File: manager/controllers/advisory_systems.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func AdvisorySystemsListHandler(c *gin.Context) {
 
 	query, err = ApplySort(c, query, SystemsSortFields...)
 	if err != nil {
-		LogAndRespError(c, err, err.Error())
+		LogAndRespBadRequest(c, err, "sort application failed")
```
Comment (semtexzv) [inline manager/controllers/advisory_systems.go]: Could we add a test case for this ? Returning invalid request on invalid sort field.

Comment (Josca) [reply manager/controllers/advisory_systems.go]: Yes, I would create ticket to increase test coverage in the next sprint. Or do you really want to add it into this PR?

Comment (semtexzv) [reply manager/controllers/advisory_systems.go]: Why not, it's a short PR

Comment (Josca) [reply manager/controllers/advisory_systems.go]: ok, test added

**Pull request merged**